python/sglang/srt/layers/triton_fused_moe/configs/README

```diff
@@ -0,0 +1,10 @@
+This directory contains tuned configurations for different settings of the fused_moe kernel.
```

Comment:
1. also copy the auto-tuned configs.
2. @ispobock you can probably auto-tune for some deepspeek models

Reply:
ref https://github.com/sgl-project/sglang/pull/2155